### PR TITLE
[13.0][MIG] sequence_check_digit

### PR DESCRIPTION
**Pull request by jarroyomorales**
Standard migration to 13.0 of sequence_check_digit. No changes

@etobella @LoisRForgeFlow

### CODE REVIEW

File: sequence_check_digit/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Copyright (C) 2017 Creu Blanca
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Check Digit on Sequences",
+    "version": "13.0.1.0.0",
+    "category": "Reporting",
+    "website": "https://github.com/OCA/server-ux",
+    "author": "Creu Blanca, " "Odoo Community Association (OCA)",
```
Comment (Saran440) [inline sequence_check_digit/__manifest__.py]:
```suggestion
    "author": "Creu Blanca, Odoo Community Association (OCA)",
```

**Review by Saran440 — APPROVED** [review]
Functional Test. LGTM :+1:

**Review by ThomasBinsfeld — APPROVED** [review]

**Review by etobella — APPROVED** [review]

**Review by chrisdec — APPROVED** [review]
functional tests

**Pull request merged**